compiler/parser/chapel.ypp

```diff
@@ -1084,6 +1089,30 @@ $$ = buildIfStmt(convertAssignmentAndWarn($2,$3,$4), $6, $8); }
 $$ = buildIfStmt(convertAssignmentAndWarn($2,$3,$4), $5, $7); }
 ;
 
+interface_stmt:
+  TINTERFACE ident_def TLP ifc_formal_ls TRP block_stmt
+  { $$ = buildChapelStmt(InterfaceSymbol::buildDef($2, $4, $6)); }
```

Comment:
I don't think there is agreement about putting the parameters always in parens in this way. The CHIP proposal supports a single implicit parameter called Self. Additionally, I don't personally know of a reason we would ever need more than one parameter. I think the language design here needs some discussion, but it doesn't have to hold up this PR. Just don't think that adding this PR means the issue is settled.

Reply:
It is issue #16966.